watson-developer-cloud/discovery-components
# fix: import individual react-virtualized components

#### What do these changes do/fix?

`rollup` is not able to tree-shake `react-virtualized` module, resulting in our build containing several RV components that we do not use.

Fix is to import only the components we need. This results in a reduction of ~130K in built size (according to `yarn analyze`). Also, it removes the `Import of non-existent export` warning.

#### How do you test/verify these changes?

Run build and `yarn analyze` before and after changes. Notice reduction in size and removal of warning.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
